src/chat/components/camera/VideoPlayer.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useState, useRef } from 'react';
+import {
+  View,
+  Text,
+  StyleSheet,
+  TouchableOpacity,
+  Image,
+  Dimensions,
+} from 'react-native';
+import Video, {
+  OnLoadData,
+  OnProgressData,
+  VideoRef,
+} from 'react-native-video';
+import { formatTime } from '../../../utilities';
+import CustomSlider from '../slider';
+
+const { height, width } = Dimensions.get('window');
+interface VideoPlayerProps {
+  videoUri: string;
+  sliderCustom?: (
+    currentTime: number,
+    duration: number,
+    paused: boolean,
+    videoRef: VideoRef | null
+  ) => React.ReactNode;
+}
+
+const ImageAssets = {
+  playIcon: require('../../../images/play_white.png'),
```

Comment:
Can you put these icons in `src/chat/asset/index.ts` too?

Reply:
ok I updated